backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/dsl/impl/AbstractElasticsearchNumericFieldTypeOptionsStep.java

```diff
@@ -65,6 +68,19 @@ protected final void complete() {
 			builder.aggregable( true );
 			builder.queryElementFactory( AggregationTypeKeys.TERMS, new ElasticsearchTermsAggregation.Factory<>( codec ) );
 			builder.queryElementFactory( AggregationTypeKeys.RANGE, new ElasticsearchRangeAggregation.Factory<>( codec ) );
+			builder.queryElementFactory( AggregationTypeKeys.SUM,
+					new ElasticsearchMetricFieldAggregation.Factory<>( codec, "sum" ) );
+			builder.queryElementFactory( AggregationTypeKeys.MIN,
+					new ElasticsearchMetricFieldAggregation.Factory<>( codec, "min" ) );
+			builder.queryElementFactory( AggregationTypeKeys.MAX,
+					new ElasticsearchMetricFieldAggregation.Factory<>( codec, "max" ) );
+			builder.queryElementFactory( AggregationTypeKeys.COUNT,
+					new ElasticsearchMetricLongAggregation.Factory<>( codec, "value_count" ) );
+			builder.queryElementFactory( AggregationTypeKeys.COUNT_DISTINCT,
+					new ElasticsearchMetricLongAggregation.Factory<>( codec, "cardinality" ) );
+			builder.queryElementFactory( AggregationTypeKeys.AVG,
+					new ElasticsearchMetricDoubleAggregation.Factory<>( codec, "avg" )
+			);
```

Comment:
Maybe let's try testing these aggregations with other types as well and see which are supported (I'd hope that at least temporal types would work with some of these). When I say test, I mean just to create an index and run some simple aggregation HTTP request so that we don't need to add all the code if the backend does not even support it 🙈 😃.